<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DaleDanT_TextGen</class>
 <widget class="QDialog" name="DaleDanT_TextGen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>311</width>
     <height>281</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>DD_Small.PNG</pixmap>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>370</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>450</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate Text!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>350</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter Your Text:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>-30</y>
     <width>531</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>Dale Dan Tony Text Generator 9000</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>530</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>510</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Your Text:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
